src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -19,15 +19,23 @@ export default function ColorPickerControl({
 }: Props): JSX.Element | null {
     const paletteRef = React.useRef<HTMLDivElement>(null);
     const [isColorPickerToggled, setIsColorPickerToggled] = React.useState(false);
-    const [isPaletteActive, handlers] = useAttention();
+    const { current: paletteEl } = paletteRef;
+
+    const handlePaletteBlur = ({ relatedTarget }: React.FocusEvent<HTMLButtonElement>): void => {
+        const targetElement = relatedTarget as HTMLButtonElement;
```

Comment:
I don't think IE11 supports `relatedTarget`, so we may need to fall back to `document.activeElement` here as in the `handleToggleBlur` function below.